File: common/protcol/c#/module/hub_call_centermodule.cs
/*this module file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.Collections.Generic;

namespace module
{
    public class hub_call_center : juggle.Imodule 
    {
        public hub_call_center()
        {
			module_name = "hub_call_center";
        }

        public delegate void closedhandle();
        public event closedhandle onclosed;
        public void closed(ArrayList _event)
        {
            if(onclosed != null)
            {
                onclosed();
            }
        }

	}
}

File: common/protcol/js/module/hub_call_centermodule.js
/*this module file is codegen by juggle for js*/
function hub_call_center_module(){
    eventobj.call(this);
    Imodule.call(this, "hub_call_center");

    this.closed = function(){
        this.call_event("closed", []);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Imodule.prototype;
    hub_call_center_module.prototype = new Super();
})();
hub_call_center_module.prototype.constructor = hub_call_center_module;

